src/features/MentorPage/components/MentorList/MentorCard/List/Header.tsx

```diff
@@ -20,18 +23,31 @@ export const Header: React.FC<Props> = ({
   age,
   region,
   isAvailable,
+  isMe,
+  isNew,
   message,
 }) => {
   const { t } = useTranslation('mentors');
   const { isMobile } = useGetLayoutMode();
 
-  const availabilityMessage = isAvailable ? '' : t('card.unavailable');
+  const getStatus = (
+    isMe: boolean,
+    isAvailable: boolean,
+    isNew: boolean,
+  ): Status => {
+    if (isMe) {
+      return 'me';
+    } else if (!isAvailable) {
+      return 'unavailable';
+    } else if (isNew) {
+      return 'new';
+    }
+    return 'empty';
+  };
```

Comment:
Instead of using `else if` we prefer to use only `if` statements in the code.
It is easier to read then. =)

So

```javascript
function(x, y){
  if(x){
    return 'foo'
  }
  if(y){
    return 'bar'
  }

  return 'baz'
}
```

Reply:
Done!